content/develop/reference/modules/modules-api-ref.md

```diff
@@ -3840,7 +3840,7 @@ AOF rewrite, and so forth). In this section we define this API.
 
 Register a new data type exported by the module. The parameters are the
 following. Please for in depth documentation check the modules API
-documentation, especially [/develop/reference/modules/modules-native-types]({{< relref "/develop/reference/modules/modules-native-types" >}}).
+documentation, especially [https://redis.io/topics/modules-native-types](https://redis.io/topics/modules-native-types).
```

Comment:
The link seems to go to the docs homepage, so presumably dodgy.

Reply:
Same issue: these links are embedded in source code, so I'll need to hunt them down and fix them there.

Fixed with next push.